features/admin.users.v1/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -304,24 +304,26 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
     };
 
     /**
-     * Check the given user store is Read/Write enabled
+     * Check the given user store is Read/Write enabled and bulk import supported.
      *
      * @param userStore - Userstore
-     * @returns If the given userstore is read only or not
+     * @returns If the given userstore is read only or not and bulk import is supported.
      */
-    const checkReadWriteUserStore = (userStore: UserStoreDetails): Promise<boolean> => {
+    const isBulkImportSupportedUserStore = (userStore: UserStoreDetails): Promise<boolean> => {
         let isReadWriteUserStore: boolean = false;
+        let isBulkImportSupported: boolean = false;
 
         return getAUserStore(userStore?.id).then((response: UserStoreDetails) => {
```

Comment:
Instead of sending separate API calls to get the properties for each userstores, we can request the properties to be directly included in the initial userstore list response. (/userstores?requiredAttributes=ReadOnly,BulkUserImportSupported)

Reply:
If you are constrained by time or scope, let's create an improvement issue to fix this later.

Reply:
Added the improvement with [14cdbcc](https://github.com/wso2/identity-apps/pull/7565/commits/14cdbcc718f793e970ac0deefa94e06839796f63). Missed this one, thanks for pointing it out.

Reply:
Thanks.